src/Implementations/ComposerProcessor.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace De\Idrinth\TestGenerator\Implementations;
+
+use De\Idrinth\TestGenerator\Interfaces\TestClassDecider as TCDI;
+use De\Idrinth\TestGenerator\Interfaces\ComposerProcessor as CPI;
+use InvalidArgumentException;
+
+class ComposerProcessor implements CPI
+{
+    /**
+     * @var TCDI
+     */
+    private $decider;
+
+    /**
+     * @var string
+     */
+    private $output;
+
+    /**
+     * @param TCDI $decider
+     * @param string $output
+     */
+    public function __construct(TCDI $decider, $output = '')
+    {
+        $this->decider = $decider;
+        $this->output = ($output?DIRECTORY_SEPARATOR:'').$output;
+    }
+
+    /**
```

Comment:
inheritdoc could be possible here, see: src/Interfaces/ComposerProcessor.php

Reply:
not going to use inheritdoc, it's way too easy for it to turn to lies because changing the doc is even further removed from where the implementation it documents is